target/linux/ramips/dts/Telco-Electronics-X1.dts

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/dts-v1/;
+
+#include "Telco-Electronics-X1.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "telco-electronics,x1", "mediatek,mt7621-soc";
+	model = "Telco Electronics X1";
+
+	memory@0 {
+		device_type = "memory";
+		reg = <0x0 0x10000000>;
+	};
+};
+
+&spi0 {
+	status = "okay";
+
+	m25p80@0 {
```

Comment:
Unless your bootloader needs this to be m25p80@0, this should be named flash@0 (See DT' spec generic name recommendation).

Reply:
flash@0 worked